### PR TITLE
Add software as outputs category

### LINKED ISSUE #194: Add outputs category for software
We have produced a custom software for ASAP including `asapdiscovery`, `argos` `choppa` and others. We should link to these in the output section of the website? 

For `asapdiscovery` we should probably transfer ownership to the `asapdiscovery` org first.

### PR DESCRIPTION
**Pull request by hmacdope**
Fixes #194

### CODE REVIEW

**Comment by hmacdope** [comment]
@jchodera @ijpulidos I think this is ready for review.

File: data/outputs/software.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+asapdiscovery:
+  name: "asapdiscovery"
+  description: Monorepo for running computational chemistry project support for ASAP targets, including docking, free energy calculations, chemoinformatics  and machine learning.
```
Comment (JenkeScheen) [inline data/outputs/software.yaml:3]:
```suggestion
  description: Monorepository for running computational chemistry project support for ASAP's viral targets, including docking, free energy calculations, chemoinformatics  and machine learning.
```

File: data/outputs/software.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  projects: [Project 5, Computational Chemistry Core]
+  links:
+  - name: GitHub
+    url: https://github.com/asapdiscovery/asapdiscovery
+  - name: Documentation
+    url: https://asapdiscovery.readthedocs.io/en/latest
+
+choppa:
+  name: "choppa"
+  description: "A Python library for visualizing DMS data and other fitness data directly on protein structures."
```
Comment (JenkeScheen) [inline data/outputs/software.yaml:14]:
```suggestion
  description: "A Python library for visualizing Deep Mutational Scanning data and other fitness data directly onto protein structures to enable fitness-informed decision-making in medicinal chemistry workflows in P2-P5 of ASAP's drug discovery pipeline."
```

File: data/outputs/software.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  permalink: http://asapdiscovery.org/outputs/software/#choppa
+  projects: [Project 5, Computational Chemistry Core]
```
Comment (JenkeScheen) [inline data/outputs/software.yaml:16]:
```suggestion
  projects: [Project 2, Project 5, Computational Chemistry Core]
```

File: data/outputs/software.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  links:
+  - name: GitHub
+    url: https://github.com/asapdiscovery/choppa
+  - name: Documentation
+    url: https://choppa.readthedocs.io
+
+argos:
+  name: "argos"
+  description: "A web based visualisation tool for viewing DMS data on protein structures."
```
Comment (JenkeScheen) [inline data/outputs/software.yaml:25]:
```suggestion
  description: "A web-based visualization tool for viewing DMS data on protein structures that allows users to generate and share these views using a web UI. See 'choppa' package below."
```

File: data/outputs/software.yaml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+  permalink: http://asapdiscovery.org/outputs/software/#argos
+  projects: [Project 5, Computational Chemistry Core]
```
Comment (JenkeScheen) [inline data/outputs/software.yaml:27]:
```suggestion
  projects: [Project 2, Computational Chemistry Core]
```

File: data/outputs/software.yaml [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+  links:
+  - name: GitHub
+    url: https://github.com/asapdiscovery/argos 
+
+
+hindsight:
+  name: "hindsight"
+  description: "Generates and continuously updates plots comparing computational chemistry predictions with experimental data to aid in lead optimization campaigns."
```
Comment (JenkeScheen) [inline data/outputs/software.yaml:35]:
```suggestion
  description: "Generates and continuously updates statistics and graphs comparing computational chemistry predictions with gathered experimental data to aid in hit-to-lead and lead optimization campaigns."
```

File: data/outputs/software.yaml [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+  permalink: http://asapdiscovery.org/outputs/software/#hindsight
+  projects: [Project 5, Computational Chemistry Core]
+  links:
+  - name: GitHub
+    url: https://github.com/asapdiscovery/hindsight-public
+
+
+falcbot:
+  name: "falcbot"
+  description: "A slack integration for running computational chemistry workflows."
```
Comment (JenkeScheen) [inline data/outputs/software.yaml:45]:
```suggestion
  description: "A slack integration for running computational chemistry workflows that are developed by ASAP's computational chemistry core."
```

**Review by JenkeScheen — CHANGES_REQUESTED** [review]
Looks great! I suggested some changes in phrasing but other than that this is spot on.

**Comment by hmacdope** [comment]
@JenkeScheen suggestions applied

**Review by JenkeScheen — APPROVED** [review]

**Pull request merged**